examples/solids/src/setuplibceed.c

```diff
@@ -37,35 +37,35 @@ problemData problemOptions[3] = {
     .setupgeo = SetupGeo,
     .apply = LinElasF,
     .jacob = LinElasdF,
-    .energy = LinElasEnergy,
+    .energy = StrainEnergy,
     .setupgeofname = SetupGeo_loc,
     .applyfname = LinElasF_loc,
     .jacobfname = LinElasdF_loc,
-    .energyfname = LinElasEnergy_loc,
+    .energyfname = StrainEnergy_loc,
     .qmode = CEED_GAUSS
   },
   [ELAS_HYPER_SS] = {
     .qdatasize = 10,
     .setupgeo = SetupGeo,
     .apply = HyperSSF,
     .jacob = HyperSSdF,
-    .energy = HyperSSEnergy,
+    .energy = StrainEnergy,
     .setupgeofname = SetupGeo_loc,
     .applyfname = HyperSSF_loc,
     .jacobfname = HyperSSdF_loc,
-    .energyfname = HyperSSEnergy_loc,
+    .energyfname = StrainEnergy_loc,
```

Comment:
Note that these aren't the correct strain energies for the linear and small-strain models. Ideally we would report the strain energy for the model actually being used. It's just a deficiency of our notes that we only explicitly wrote down the energy for the Neo-Hookean model.

Although not critical for our scaling studies, our future trajectory will have alternatives to Neo-Hookean.
Computation of strain energy shouldn't be performance sensitive so it's okay to factor however is most convenient.

The repetition in certain `static inline` ingredients is a reason why it'd be useful for qfunction source files to be allowed to include headers. We can discuss that after #305 merges.

Reply:
Agreed on all fronts. I am being very messy in my rush to get these convergence studies done. I anticipate rebuilding this branch after I distribute my proposal.

Reply:
I'd like to get the bug fix into master sooner rather than later. If you report at "Neo-Hookean Strain Energy" then I'd be fine with merging it and we can make it match the model later.

Reply:
I made a clean PR with just the fixes.